src/pesterTests.ts

```diff
@@ -53,6 +54,20 @@ export class PesterTestRunner {
 	
 		const exePath = await this.getPowerShellExe();
 
+		const pesterVersionScript = getPesterVersion();
+		this.log.debug(pesterVersionScript);
+		const pesterVersion = spawn(exePath, [
```

Comment:
What happens if the user doesn't have Pester installed? Since you're here, would you mind maybe checking for that as well?

Reply:
I agree, the order of operations should be considered here (e.g. check that Pester is installed before checking the Pester module version), however, I don't think this Pull Request should include such a significant change.  I feel that magnitude of a change should be its own PR (or User Story).  I'm not expecting a new release of the extension based on this P.R., I would expect that a collection of updates (features & bug fixes) be considered for a new release.

Additionally, there are other sections of code that check for the Pester module and even attempt to install the Pester module for the end user.  (see "function GetPesterInvokeScript" in `powershellScripts.ts`)  Personally, I don't agree with installing modules "under the hood" for the end user, I wouldn't want a program updating dependencies without explicit warning / foreknowledge.

That said -- I will continue work to improve this process and address the required dependency checks.